src/components/UpcomingShiftCard/UpcomingShiftCard.css

```diff
@@ -0,0 +1,18 @@
+.card{
+  box-shadow: rgba(0, 0, 0, 0.25) 0px 1px 4px;
+  width: 400px;
+  border: none;
+}
+.card-body{
+  padding: 15px 20px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+.card-title{
+  margin: 0;
+}
+.card-link{
+  color: #6666FF;
+  font-weight: lighter;
+}
```

Comment:
Delete this file I don't think you need it anymore because you're using Bootstrap for your component.